hybrid/decomposers.py

```diff
@@ -36,6 +37,9 @@
     bqm_induced_by, flip_energy_gains, select_random_subgraph,
     chimera_tiles)
 
+from networkx.algorithms.approximation.vertex_cover import min_weighted_vertex_cover
+from dwave.samplers import SteepestDescentSolver, TreeDecompositionSolver
```

Comment:
It's important, thanks for pointing out these errors

Reply:
The tests are currently failing because `dwave-samplers` is not a dependency (instead we use the separate neal/greedy/etc packages). I'll refresh the deps in a separate PR.